content/manager_webui/logs-events-page.md

```diff
@@ -3,38 +3,88 @@ layout: bt_wiki
 
-You can control which events are displayed by filtering by:
+### Deployments
+To filter logs & events by specific deployments choose one or more deployments, matching logs & events will be automatically updated.<br/>
+![deployments filter]({{< img "ui/eventsPage/deploymentsFilter.png" >}})<br>
 
-# Blueprints
-See the definition [here]({{page.terminology_link}}#blueprint).<br/>
-![Blueprints]({{< img "ui/ui-logsNevents-blueprint-selection.png" >}})
+### Log Levels
+To filter logs & events by specific log levels choose one or more log levels, matching logs & events will be automatically updated.<br/>
+![log levels filter]({{< img "ui/eventsPage/loglevelsFilter.png" >}})<br>
+
+### Event Types
+To filter logs & events by specific event types choose one or more event types, matching logs & events will be automatically updated.<br/>
+![event types filter]({{< img "ui/eventsPage/eventTypesFilter.png" >}})<br>
+
+### Time Range
+To filter logs & events from and to a specific timestamp use the time range filters.
+The filters consist of 2 timepickers, each allow manually typing the timestamp expected to be in `YYYY-MM-DD HH:MM` format or choosing a date and time from the timepicker.<br>
+The first timepicker filter logs & events `from` the chosen timestamp, the second filter log & events `to` the chosen timestamp.
+You can choose to filter using either of the timepickers or both.
+![time range filters]({{< img "ui/eventsPage/timeRangeFilter.png" >}})<br>
+
+
+{{% gsNote title="Note" %}}
+If it is unclear is your time is valid, look for this <i class="fa fa-calendar"></i> icon. When it shows - the input is valid and the data is automatically changed to match the filter.<br>
+![time range]({{< img "ui/eventsPage/timeRange.png" >}})<br>
+{{% /gsNote %}}
+
+### Message Text
+To filter logs & events by specific message text free type the message, matching logs & events will be automatically updated.<br/>
+![message text filter]({{< img "ui/eventsPage/messageTextFilter.png" >}})<br>
+
+### Clear Filters
+To clear all of the filters you can easily do so by clicking the 'Clear Filters' button.
+
+# Sorting
+To sort logs & events click on the desired table header to switch order between ascending, descending and none.
+![sorting]({{< img "ui/eventsPage/sorting.png" >}})<br>
+
+# Pagination
+50 logs & events are presented per page. You can use the pagination bar to scroll through the pages.
+![pagination]({{< img "ui/eventsPage/pagination.png" >}})<br>
+
+# The Log/Event
+By default the table is not presenting all of the log/event information. In order to reveal all of the log/event information simply click on the row itself.<br>
+An event example:
+![event example]({{< img "ui/eventsPage/event.png" >}})<br>
+
+A log example:
+![log example]({{< img "ui/eventsPage/log.png" >}})<br>
+
+{{% gsNote title="Note" %}}
+Not all logs/events has the same fields and there are some fields which are specific to a type.<br>
```

Comment:
regarding type- meant type as log/event.
As even you were confused about it , it is obviously not good enough for docs.. so rephrased to this:
"Not all logs/events have the same fields as there are some fields which are specific to logs and others which are specific to events."